rs/sns/test_utils/src/itest_helpers.rs

```diff
@@ -1145,14 +1149,18 @@ impl SnsCanisters<'_> {
     /// Await a Proposal being rewarded. This is detected by inspecting it's
     /// reward_event_end_timestamp_seconds field (detection used to be via the
     /// reward_round_event field, but that is now deprecated).
-    pub async fn await_proposal_rewarding(&self, proposal_id: ProposalId) -> ProposalData {
+    pub async fn await_proposal_rewarding(
```

Comment:
I kind of feel like SnsCanisters should just keep a reference to Runtime, since it's already got it in the canisters it holds... lifetimes shouldn't be a big issue.  Then this would look a lot nicer when you use it in tests...

Reply:
I agree that would be an improvement.